flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BucketWriter.java

```diff
@@ -516,12 +504,12 @@ public synchronized void append(final Event event)
             consecutiveUnderReplRotateCount >= maxConsecUnderReplRotations) {
           doRotate = false;
           if (consecutiveUnderReplRotateCount == maxConsecUnderReplRotations) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)